create_image.sh

```diff
@@ -14,8 +14,14 @@ fi
 sudo yum install -y java-1.7.0-openjdk-devel gcc gcc-c++ ant git
 # Perf tools
 sudo yum install -y dstat iotop strace sysstat htop perf
+
+sudo yum-complete-transaction ; sleep 5
```

Comment:
Sometimes the build fails here because `yum` holds a lock and doesn't release it quickly enough. I've tried various things to identify or get around this problem, but I still haven't gotten to the bottom of it.

This line may or may not be helping with that. I won't be sure until I run the build a whole bunch more times, since the original problem was intermittent.